dev/workflow/rocoto/gfs_tasks.py

```diff
@@ -741,6 +741,30 @@ def marinebmat(self):
             deps.append(rocoto.add_dependency(dep_dict))
         dependencies = rocoto.create_dependency(dep_condition='and', dep=deps)
 
+        resources = self.get_resource('marinebmatinit')
+        task_name = f'{self.run}_marinebmatinit'
+        task_dict = {'task_name': task_name,
+                     'resources': resources,
+                     'dependency': dependencies,
+                     'envars': self.envars,
+                     'cycledef': self.run.replace('enkf', ''),
+                     'command': f'{self.HOMEgfs}/dev/jobs/marinebmatinit.sh',
+                     'job_name': f'{self.pslot}_{task_name}_@H',
+                     'log': f'{self.rotdir}/logs/@Y@m@d@H/{task_name}.log',
+                     'maxtries': '&MAXTRIES;'
+                     }
+
+        task = rocoto.create_task(task_dict)
+
+        return task
+
+    def marinebmat(self):
+
+        deps = []
+        dep_dict = {'type': 'task', 'name': f'{self.run}_marinebmatinit'}
+        deps.append(rocoto.add_dependency(dep_dict))
+        dependencies = rocoto.create_dependency(dep_condition='and', dep=deps)
```

Comment:
```suggestion
        dependencies = rocoto.create_dependency(dep=deps)
```
There is no other dependency, `and` is unnecessary